src/user-guide/running-workflows/tracking-task-state.rst

```diff
@@ -123,33 +123,49 @@ that do not allow TCP or non-interactive SSH from job host to workflow host.
 Be careful to avoid spamming task hosts with polling operations. Each poll
 opens (and then closes) a new SSH connection.
 
-Polling intervals are configurable under :cylc:conf:`[runtime]` because they
-may depend on expected job execution time. You may want to poll a job
-frequently at first, to check that it started running properly; frequently
-near the expected end of its run time, to get a timely task finished update;
-and infrequently between times. Configured intervals are used in sequence until
+Polling intervals are configurable under
+:cylc:conf:
+`global.cylc[platforms][<platform name>]submission polling intervals`
+and
+:cylc:conf:
+`global.cylc[platforms][<platform name>]execution polling intervals`
+Under normal conditions you should be able to use site configuration
+for any given platform.
+
+.. include:: ../../reference/config/user-platform-warning.rst
```

Comment:
Not sure about this

```suggestion
```

Reply:
I've removed the first paragraph and reworded the second.